### PR TITLE
v2

### PR DESCRIPTION
**Pull request by blacksails**
- Update sonarqube documentation
- Initial implementation

### CODE REVIEW

File: .github/workflows/master.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        with:
+          path: ~/.npm
+          key: npm-${{ hashFiles('package-lock.json') }}
+          restore-keys: |
+            npm-
+      - run: |
+          npm ci
+          npm run build
+          npm run test
+      - uses: tradeshift/actions-sonarqube/scanner@v1
```
Comment (sorenmat) [inline .github/workflows/master.yml]: remove?

File: .github/workflows/master.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            npm-
+      - run: |
+          npm ci
+          npm run build
+          npm run format-check
+          npm run lint
+          npm run package
+          npm run test
+
+  sonarqube:
```
Comment (sorenmat) [inline .github/workflows/master.yml]: remove, consider running the github super linter

File: __tests__/main.test.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+test('todo', async () => {});
```
Comment (sorenmat) [inline __tests__/main.test.ts:1]: image

Comment (blacksails) [reply __tests__/main.test.ts:1]: I det mindste er der 1 test som test at ingenting virker

Comment (blacksails) [reply __tests__/main.test.ts]: I added tests for argument creation now

**Review by sorenmat — APPROVED** [review]
LGTM

**Pull request merged**